src/pages/docs/kubernetes/workers/storage.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2024-04-29
+modDate: 2024-07-31
+title: Storage
+description: How to configure storage for a Kubernetes worker
+navOrder: 30
+---
+
+During a deployment, Octopus Server first sends any required scripts and packages to [Tentacle](https://octopus.com/docs/infrastructure/deployment-targets/tentacle) which writes them to the file system. The actual script execution then takes place in a different process called [Calamari](https://github.com/OctopusDeploy/Calamari), which retrieves the scripts and packages directly from the file system. 
+
+On a Kubernetes agent, scripts are executed in separate Kubernetes pods (script pod) as opposed to in a local shell (Bash/Powershell). This means the Tentacle pod and script pods don’t automatically share a common file system.
+
+Since the Kubernetes agent is built on the Tentacle codebase,  it is necessary to configure shared storage so that the Tentacle Pod can write the files in a place that the script pods can read from.
+
+We offer two options for configuring the shared storage - you can use either the default NFS storage or specify a Storage Class name during setup:
+
+:::figure
+![Kubernetes Agent Wizard Config Page](/docs/infrastructure/workers/images/kubernetes-worker-wizard-config.png)
+:::
+
+
+## NFS storage
+
```

Comment:
Should we recommend using NFS?

Reply:
file has been removed - leaning into the agent's docs